### PR TITLE
[Backup] Monthly reminder

### PR DESCRIPTION
**Pull request by flypaper0**

### CODE REVIEW

File: VultisigApp/VultisigApp/Views/Vault/VaultDetailView.swift [hunk 1]
```diff
@@ -180,6 +204,16 @@ struct VaultDetailView: View {
             .listRowInsets(EdgeInsets())
             .listRowSeparator(.hidden)
     }
+
+    private func showMonthlyReminderIfNeeded() {
+        guard !vault.isBackedUp else { return }
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/Views/Vault/VaultDetailView.swift:209]: regardless whether a vault has been backup before , we always need to remind user monthly

**Review by johnnyluo — CHANGES_REQUESTED** [review]
can you please remove `!vault.isBackedUp` check?

**Comment by flypaper0** [comment]
> can you please remove `!vault.isBackedUp` check?

why we should? It make sense to me to not show it when vault is backed up

File: VultisigApp/VultisigApp/Views/Vault/VaultDetailView.swift [hunk 2]
```diff
@@ -152,7 +171,12 @@ struct VaultDetailView: View {
         DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
             isLoading = false
         }
+        
         setData()
+
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
```
[the remark this answers is not in the document]
Comment (flypaper0) [reply VultisigApp/VultisigApp/Views/Vault/VaultDetailView.swift:177]: it should, placed breakpoint, In 10 from 10 runs it was called

**Comment by johnnyluo** [comment]
> > can you please remove `!vault.isBackedUp` check?
> 
> why we should? It make sense to me to not show it when vault is backed up

Even user did their backup , they can forget where they put it , or lost it or whatever , this remind will trigger them to double check , or maybe backup it again .

**Comment by flypaper0** [comment]
> > > can you please remove `!vault.isBackedUp` check?
> > 
> > 
> > why we should? It make sense to me to not show it when vault is backed up
> 
> Even user did their backup , they can forget where they put it , or lost it or whatever , this remind will trigger them to double check , or maybe backup it again .

ok, I see

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**